css/types/color.json

```
@@ -1051,6 +1051,43 @@
               "deprecated": false
             }
           },
+          "accentcolor_accentcolortext": {
+            "__compat": {
+              "description": "<code>AccentColor</code> and <code>AccentColorText</code>",
+              "spec_url": [
+                "https://drafts.csswg.org/css-color/#valdef-system-color-accentcolor",
+                "https://drafts.csswg.org/css-color/#valdef-system-color-accentcolortext"
+              ],
+              "support": {
+                "chrome": {
+                  "version_added": false
```

flags can be set like this:

```suggestion
                  "version_added": false,
                  "flags": [
                    {
                      "type": "preference",
                      "name": "#enable-experimental-web-platform-features",
                      "value_to_set": "true"
                    }
                  ]
```